trpcultivate_phenotypes/src/Plugin/Validators/validDataFile.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/**
+ * @file
+ * Contains validator plugin definition.
+ */
+
+namespace Drupal\trpcultivate_phenotypes\Plugin\Validators;
+
+use Drupal\trpcultivate_phenotypes\TripalCultivateValidator\TripalCultivatePhenotypesValidatorBase;
+use Drupal\trpcultivate_phenotypes\TripalCultivateValidator\ValidatorTraits\FileTypes;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\file\Entity\File;
+
+/**
+ * Validate data file.
+ *
+ * @TripalCultivatePhenotypesValidator(
+ *   id = "trpcultivate_phenotypes_validator_valid_data_file",
+ *   validator_name = @Translation("Valid Data File Validator"),
+ *   input_types = {"file"}
+ * )
+ */
+class ValidDataFile extends TripalCultivatePhenotypesValidatorBase implements ContainerFactoryPluginInterface {
```

Comment:
I noticed that the filename for this class starts as lowercase instead of uppercase. I don't know how to suggest a change like that through Github though so I'm mentioning it here 😆

Reply:
Git did not allow to push just a filename change but did allow with a bit of code changes plus the filename change.